.travis.yml

```diff
@@ -30,3 +33,8 @@ script:
 
 after_failure:
     - for i in $(find tests -name \*.actual); do echo "--- $i"; cat $i; echo; echo; done
+
+cache:
+  directories:
+    - vendor
```

Comment:
Necachuje se composer.lock, takze by to melo provest dohledani balicku, dependency graph a vlastne vse krome vlastniho stazeni kodu balicku.

Ale mas pravdu ze je dobre videt v logu verze knihoven a zisk cachovanim vendor slozky nebude podstatny.

Force pushnul jsem zmeny.